UnitTests/GitUI.Tests/Avatars/InitialsAvatarGeneratorTests.cs

```diff
@@ -10,36 +10,22 @@ namespace GitUITests.Avatars
     [TestFixture]
     public sealed class InitialsAvatarGeneratorTests
     {
-        [Test]
-        public void GetInitialsAndHashCode_return_initials_of_a_user()
+        [TestCase("albert.einstein@noreply.com", "albert einstein", "AE")]
+        [TestCase("albert.einstein@noreply.com", "albert z middlename einstein (k)", "AE")]
+        [TestCase("albert.einstein@noreply.com", "albert einstein (Nobel_price_winner)", "AE")]
+        [TestCase("albert.einstein@noreply.com", "EINSTEIN, Albert middlename (Nobel_price_winner)", "EM")]
+        [TestCase("albert.einstein@noreply.com", "albert", "AL")]
+        [TestCase("albert.einstein@noreply.com", "", "AE")]
+        [TestCase("albert.einstein@noreply.com", null, "AE")]
+        [TestCase("albert.z.einstein@noreply.com", null, "AE")]
+        [TestCase("albert", "", "AL")]
```

Comment:
Perhaps in this case, it's better to have the 2nd letter lowercase to indicate that it was not the first letter of a name...